portfolio/src/main/webapp/style.css

```diff
@@ -93,11 +93,16 @@ body {
   background-color: var(--quaternary);
 }
 
+#intro {
+  background: var(--primary);
+}
+
 #intro,
 .intro-wrapper {
   align-items: center;
   display: flex;
   justify-content: center;
+  color:white;
```

Comment:
Nit: You might want to sort this in alphabetically as well.

Reply:
Thanks, alphabetised the attributes with commit 48d2ccf